.github/workflows/lint-chart.yaml

```diff
@@ -3,9 +3,9 @@ name: Chart Lint
 
 env:
   HELM_VERSION: v3.11.2
-  KIND_VERSION: v0.14.0
-  KIND_NODE_IMAGE: kindest/node:v1.27.3
-  K8S_VERSION: v1.26.4
+  KIND_VERSION: v0.20.0
+  KIND_NODE_IMAGE: kindest/node:v1.29.0
+  K8S_VERSION: v1.29.0
```

Comment:
This PR would not trigger this CI, so i have standalone trigger CI for this change and CI is green, see: https://github.com/karmada-io/karmada/actions/runs/7308593108/job/19915359313